### ISSUE TITLE
Learn How to Talk About a Typical Workday and Tasks - Task 37 - typing cafés

### ISSUE THREAD

**Issue by gikf** [freeCodeCamp/freeCodeCamp#53771]
**Describe the bug**
Task 37 requires typing `cafés` in one of the blanks. This can be problematic due to the `é` letter, which might not be present in the used keyboard layout.

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/657e2c02a685f4ec9ec951ff.md

**Expected behavior**
I don't want to say to use completely different word for the _fill the blank_, as at some point there might be no other choice. There could be some kind way to ease up procuring of the letter in question, for example including it in the feedback, so it can be copied.

**Comment by shivasaikrishnagadeela** [comment]
Hi @gikf I would love to work on this. If someone could point me in the right direction, that would be amazing!

**Comment by gikf** [comment]
Hi @shivasaikrishnagadeela,

Thank you for being eager to contribute.

This issue has not been triaged yet. Feel free to share your insights and ideas on the matter. However as far as the code contributing, take a look at the issues with `help wanted` label.

Please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

Thank you and happy coding.

**Comment by naomi-lgbt** [comment]
Yes, I think in both of those challenges we can update the "Don't forget the stress mark." in the feedback to:

"Don't forget the stress mark (`é`)."

**Comment by ilenia-magoni** [comment]
> "Don't forget the stress mark (`é`)."

I will open the issue to contribution with this as the solution
